packages/data-context/src/actions/ElectronActions.ts

```diff
@@ -107,12 +106,16 @@ export class ElectronActions {
     })
   }
 
-  showSystemNotification (title: string, body: string) {
-    const notification = this.ctx.electronApi.createNotification(title, body, !this.isMac ? getPathToIcon('icon_64x64.png') : undefined)
+  showSystemNotification (title: string, body: string, onClick?: () => void) {
+    const notification = this.ctx.electronApi.createNotification(title, body)
 
-    notification.on('click', async () => {
+    const defaultOnClick = async () => {
       await this.ctx.actions.browser.focusActiveBrowserWindow()
-    })
+    }
+
+    const clickHandler = onClick || defaultOnClick
```

Comment:
/nit: Could avoid building this function if an `onClick` was provided to save a processor cycle:

```js
const clickHandler = onClick || (async () => {
  await this.ctx.actions.browser.focusActiveBrowserWindow()
})
```